core/src/test/java/org/mobilitydata/gtfsvalidator/validator/DefaultValidatorProviderTest.java

```diff
@@ -35,13 +35,22 @@ public void testCreateValidators() throws ValidatorLoaderException {
     GtfsTestTableContainer tableContainer =
         new GtfsTestTableContainer(TableStatus.PARSABLE_HEADERS_AND_ROWS);
     GtfsFeedContainer feedContainer = new GtfsFeedContainer(ImmutableList.of(tableContainer));
-
+    List<Class<? extends SingleEntityValidator>> singleEntityValidatorsWithParsingErrors =
+        new ArrayList<>();
     assertThat(
-            provider.createSingleEntityValidators(GtfsTestEntity.class).stream()
+            provider
+                .createSingleEntityValidators(
+                    GtfsTestEntity.class, singleEntityValidatorsWithParsingErrors::add)
+                .stream()
                 .map(Object::getClass))
         .containsExactly(GtfsTestEntityValidator.class);
 
-    assertThat(provider.createSingleFileValidators(tableContainer).stream().map(Object::getClass))
+    List<Class<? extends FileValidator>> validatorsWithParsingErrors = new ArrayList<>();
+    assertThat(
+            provider
+                .createSingleFileValidators(tableContainer, validatorsWithParsingErrors::add)
+                .stream()
+                .map(Object::getClass))
         .containsExactly(GtfsTestFileValidator.class);
 
```

Comment:
You should consider adding tests to the `testCreateValidators_skippedValidators` method below, to test the cases where there are validatorsWithParsingErrors

Reply:
I went thru your commit and it looks good to me. I like the method comments - pretty helpful. Also, the printed message "Some validators were skipped because the GTFS files they rely on could not be parsed" is much clearer!
You can go ahead merge it.